http_service/listener.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Could you move the code into `http_service/bugbug_http`?
Then we can add the script to `console_scripts` in `http_service/setup.py` so we can run it directly as `bugbug-http-pulse-listener` instead of `/code/http_service/listener.py`

Reply:
(This would also probably avoid the `pulse` addition in `.isort.cfg`).